spec/text-editor-spec.coffee

```diff
@@ -4018,3 +4018,71 @@ describe "TextEditor", ->
       editor.setPlaceholderText('OK')
       expect(handler).toHaveBeenCalledWith 'OK'
       expect(editor.getPlaceholderText()).toBe 'OK'
+
+  describe 'Gutters', ->
+    describe 'Add/Remove Gutters', ->
```

Comment:
I'd prefer to see 2 describes at this level, one for `"::addGutter"` and one for `"when a gutter is destroyed"`.